worldengine/hdf5_serialization.py

```diff
@@ -0,0 +1,220 @@
+import h5py
+from worldengine.version import __version__
+from worldengine.biome import biome_name_to_index, biome_index_to_name
+from worldengine.world import World, Step
+import numpy
+
+
+def save_world_to_hdf5(world, filename):
+    f = h5py.File(filename, libver='latest', mode='w')
+
+    general_grp = f.create_group("general")
+    general_grp["worldengine_version"] = __version__
+    general_grp["name"] = world.name
+    general_grp["width"] = world.width
+    general_grp["height"] = world.height
+
+    elevation_grp = f.create_group("elevation")
+    elevation_ths_grp = elevation_grp.create_group("thresholds")
+    elevation_ths_grp["sea"] = world.elevation['thresholds'][0][1]
+    elevation_ths_grp["plain"] = world.elevation['thresholds'][1][1]
+    elevation_ths_grp["hill"] = world.elevation['thresholds'][2][1]
+    elevation_data = elevation_grp.create_dataset("data", (world.height, world.width), dtype=float)
+    elevation_data.write_direct(world.elevation['data'])
+
+    plates_data = f.create_dataset("plates", (world.height, world.width), dtype=int)
```

Comment:
I don't know. Unless the tests fail, this will be fine. (Can you run the tests on Python 2 and 3?)